npc/quests/quests_ein.txt

```diff
@@ -1049,927 +1244,149 @@ einbroch,215,180,6	script	Uwe Kleine#ein	4_M_03,{
 				mes "and people to see. Ah well~";
 				close;
 			}
-		}
-		else if (ein_cook < 10) {
+		} else if (ein_cook < 10) {
 			mes "[Uwe]";
 			mes "Oh! Hello again,";
 			mes "you cutie adventurer.";
 			mes "So tell me, what brings";
 			mes "you here, "+strcharinfo(PC_NAME)+"?";
 			next;
-			switch(select("Ask about Cooking.", "Ask about Einbroch.", "Cancel.")) {
+			switch (select("Ask about Cooking.", "Ask about Einbroch.", "Cancel.")) {
 			case 1:
+				if (ein_cook % 2 == 0)
+					close; //Yes, there's really nothing defined for this in the Aegis Script
```

Comment:
Yes, there's no msg, no logic at all defined for this in the Aegis Script it falls through the end of the script, in the previous version for herc it would fall into case 2 which is about einbroch, Aegis script had a break to fallthrough to the end of script.